docs/source/examples/workspace/PinFile.dummy.yml

```diff
@@ -5,4 +5,4 @@ dummy-new:
 
 dummy-topo:
   topology: dummy-new.yml
-  layout: dummy.yml
+  layout: template-inventory.yml
```

Comment:
Is there a reason this changed?

Reply:
I just changed it to get more verbose info when I was working, but I didn't mean to commit it. I'll change it back